website/core/utils.py

```diff
@@ -89,11 +89,11 @@ def load_data(path):
     data = {}
     for row in rows:
         entry_type = row['Type']
-        state = row['state']
+        state = row['State']
```

Comment:
phone died. sorry

Reply:
oh lol I just realized I was talking to nobody haha